crypto/rsa/rsa_sp800_56b_check.c

```diff
@@ -289,6 +289,11 @@ int ossl_rsa_sp800_56b_check_public(const RSA *rsa)
         return 0;
 
     nbits = BN_num_bits(rsa->n);
+    if (nbits > OPENSSL_RSA_MAX_MODULUS_BITS) {
```

Comment:
I remember seeing another comment somewhere about if this upper bound is too high (which still results in large times for smaller values than this bound)

Reply:
It is not too high for this particular check and I do not want to change this value or pick anything arbitrary here.

Reply:
Did you time it?

Reply:
A few seconds on my laptop.